### PR TITLE
[T4A7][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Use TDD

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready to review

File: src/seedu/addressbook/data/person/NameTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    @Test
+    public void isSimilar_exactName_returnsTrue() throws IllegalValueException {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/NameTest.java]: hey! what happened to the `isSimilar` method that you are supposed to implement for this exercise? i see the tests but no method?

**Comment by okkhoy** [comment]
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

Going forward, we may not review PRs that are not clean.

**Comment by okkhoy** [comment]
- WRT, TDD, please look at the handout and the suggested readings in the assignment description.
- I hope the tutorial discussions and the PR we reviewed gave an idea on how to start off with TDD.

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**